client/my-sites/plans/jetpack-plans/product-store/hooks/use-store-item-info.tsx

```diff
@@ -256,9 +256,14 @@ export const useStoreItemInfo = ( {
 		( item: SelectorProduct ) => {
 			const shouldShowCart = getShouldShowCart( item );
 			const isProductInCart = getIsProductInCart( item );
+			const isOwned = getIsOwned( item );
```

Comment:
There is `productButtonLabel` used in the `getCtaLabel` function, that seems to handle this `isOwned` state etc. Should that be utilised for the lightbox too?

Reply:
This is resolved now!